### ISSUE TITLE
search replace 

### ISSUE THREAD

**Issue by kingpinzs** [microsoft/vscode#169408]
Type: <b>Bug</b>

when I do a ctrl-f and run search replace. When I put in a & to search for and a & with a new line it crashes visual studio code and once and a while my whole os

VS Code version: Code 1.74.0 (5235c6bb189b60b01b1f49062f4ffa42384f8c91, 2022-12-05T16:47:18.476Z)
OS version: Linux x64 6.0.6-76060006-generic
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 9 3900X 12-Core Processor (24 x 4137)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|11, 5, 3|
|Memory (System)|31.29GB (18.32GB free)|
|Process Argv|--unity-launch --crash-reporter-id 1a2e262e-cb1c-4ce2-b496-87403f125e7a|
|Screen Reader|yes|
|VM|0%|
|DESKTOP_SESSION|pop|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|pop|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (41)</summary>

Extension|Author (truncated)|Version
---|---|---
language-x86-64-assembly|13x|3.0.0
vscode-intelephense-client|bme|1.8.2
compare-view|che|0.10.0
vscode-eslint|dba|2.2.6
prettier-vscode|esb|9.10.3
theme-editor|imP|1.9.0
vscode-docker|ms-|1.23.2
isort|ms-|2022.8.0
python|ms-|2022.20.1
vscode-pylance|ms-|2022.12.20
jupyter|ms-|2022.11.1003412109
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.12
vscode-jupyter-cell-tags|ms-|0.1.6
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.266.1
remote-ssh|ms-|0.94.0
remote-ssh-edit|ms-|0.84.0
remote-wsl|ms-|0.72.0
vscode-remote-extensionpack|ms-|0.23.0
cmake-tools|ms-|1.12.27
cpptools|ms-|1.13.8
cpptools-extension-pack|ms-|1.3.0
makefile-tools|ms-|0.6.0
powershell|ms-|2022.12.1
remote-explorer|ms-|0.0.3
vsliveshare|ms-|1.0.5788
vsliveshare-audio|ms-|0.1.91
vsliveshare-pack|ms-|0.4.0
gremlins|nho|0.26.0
format-html-in-php|rif|1.7.0
nunjucks|ron|0.3.1
markdown-preview-enhanced|shd|0.6.5
code-spell-checker|str|2.12.0
best-light-themes-pack|the|1.0.2
cmake|twx|0.0.17
theme-generator|use|0.0.8
php-debug|xde|1.29.1
php-pack|xde|1.0.3
linkerscript|Zix|1.0.2
php-intellisense|zob|1.0.13

(11 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
pyindex848cf:30577861
nodejswelcome1:30587005
282f8724:30602487
gswce1:30612156
iaj6b796:30613358
89544117:30613380
fim-prod:30623723

```

</details>

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.74.1. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by kingpinzs** [comment]
Updated to the latest version still same issue

**Comment by andreamah** [comment]
Which find widget is it? Is it the find within a file or globally across the workspace?

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's missing some basic information or in some other way doesn't follow our [issue reporting guidelines](https://aka.ms/vscodeissuereporting). Please take the time to review these and update the issue.

Happy Coding!

**Comment by kingpinzs** [comment]
It's within a file

**Comment by rebornix** [comment]
@kingpinzs are you searching against a large file? Can you please share the document you search if possible to help troubleshoot?

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's missing some basic information or in some other way doesn't follow our [issue reporting guidelines](https://aka.ms/vscodeissuereporting). Please take the time to review these and update the issue.

Happy Coding!

**Comment by kingpinzs** [comment]
I used this text in a new file `some text & some more text & and then some more & what the & is going on ` and when I did the above steps it crashed my system.

**Comment by rebornix** [comment]
It's pure javascript code but not sure how this can crash the system. Assigning to our Electron experts.

**Comment by rzhao271** [comment]
Hi, could you attach a [crash dump](https://github.com/microsoft/vscode/wiki/Native-Crash-Issues)?

**Comment by kingpinzs** [comment]
> Hi, could you attach a [crash dump](https://github.com/microsoft/vscode/wiki/Native-Crash-Issues)?

It is not creating a dump file. Any suggestions to getting crash data from linux (Pop!_OS)

**Comment by deepak1556** [comment]
Can you please share a screen recording of the steps you have tried to get the crash dumps, thanks!

**Comment by meganrogge** [comment]
cannot repro, but would be good if @kingpinzs also verifies this is fixed

**Comment by kingpinzs** [comment]
I used the latest insiders and it worked as expected. Thank you very much! I tired it on the current release to make sure it was not fixed by me updating my system. The current release still crashed.

**Comment by kingpinzs** [comment]
Also to note in current release I disabled all plugins and it still crashed and anytime I do it twice it crashes my whole system. So it is definitely fixed in the insider release
